src/js/components/List/List.js

```diff
@@ -400,6 +460,8 @@ const List = React.forwardRef(
                 } else if (Array.isArray(adjustedBackground)) {
                   adjustedBackground =
                     adjustedBackground[index % adjustedBackground.length];
+                } else if (isPinned) {
+                  adjustedBackground = 'light-2';
```

Comment:
this really needs to be from the theme

Reply:
That makes total sense. I was having trouble understanding where the theme information was being set since it's coming from Styled Components. Do you know the file I should add this to or a reference page I can use as a resource?

Reply:
in `src/js/themes/base.js` are the base theme definitions for grommet. Other themes like the grommet-theme-hpe theme can have a sparse object of overrides as needed. So for this, if you look at about line 1023 there is a section for `list` which is where the above code gets a couple things related to disabled list items. You could have a `pinned` part below the section for disabled that has this so something like:
```jsx
pinned: {
  color: 'background-contrast',
},
```
I didn't put `light-2' there because I don't think it's very light mode versus dark mode friendly. I think in this case `background-contrast` is more appropriate (it's mainly just an opacity difference over the existing background so it makes it darker in the light theme and lighter in the dark theme. Others may weigh in on this idea.

Reply:
Awesome thank you for the detailed information. I'll review this tonight and make the appropriate changes.